_data/events.yml

```diff
@@ -24,3 +24,12 @@
   end_time: 2017-06-02 21:00
   url: https://www.facebook.com/events/1694671104175474/
   blurb: In an era where AI and virtual reality would be the next big truth, how do we make sure that this future is inclusive and reflective of everyone's realities? Joining us on Friday to talk more about this is Jasmine from Hidden Pockets.
+
+- name: nagpur-techies-june-tech-meetup-xamarin-hands-on-workshop
+  title: Nagpur Techies - June tech meetup - Xamarin hands-on workshop
```

Comment:
This might be too long a name. We need to work out a format for such events (like a weekly meet vs a one off)

For now, can we shorten this to "Xamarin hands-on workshop - Nagpur Techies"?

#4 is the issue we are tracking this on.